apps/betterangels/src/app/(private-screens)/add-note/DateAndTime/index.tsx

```diff
@@ -0,0 +1,142 @@
+import { useMutation } from '@apollo/client';
```

Comment:
this isn't part of AC, but I think it would be much easier to do here than make a new ticket. Anywhere we're hitting the `createNote` mutation, which I think it just in two places:
- `libs/expo/betterangels/src/lib/ui-components/ClientCardModal.tsx`
- `libs/expo/betterangels/src/lib/screens/Clients/index.tsx`

we should remove title from the payload
```javascript
      const { data } = await createNote({
        variables: {
          data: {
            title: `Session with ${firstName || 'Client'}`,         <-------------
            purpose: `Session with ${firstName || 'Client'}`,
            client: id,
          },
        },
      });
```

Reply:
will through and error as it is required

Reply:
`title` still required by backend. will be done in subsequent PR https://betterangels-group.slack.com/archives/C05RETRGDA7/p1728062026786269?thread_ts=1727981978.184179&cid=C05RETRGDA7